deploy/azureDeploy.js

```diff
@@ -14,13 +15,18 @@ class AzureDeploy {
 
     Object.assign(
       this,
+      createEventsBindings,
       loginToAzure,
       cleanUpFunctions,
       CreateResourceGroupAndFunctionApp,
       createFunctions
     );
 
     this.hooks = {
+      'before:package:initialize': () => BbPromise.bind(this)
```

Comment:
@lostintangent this PR doesn't allow you to call `sls package` explicitly, it just uses default one from Serverless. PR #79 has `sls package` implemented. We could implement it if needed.